src/util/ecdsa.rs

```diff
@@ -22,13 +22,17 @@ use prelude::*;
 use core::{ops, str::FromStr};
 use core::fmt::{self, Write as _fmtWrite};
 use io;
+#[cfg(feature = "std")]
+use std::error;
```

Comment:
nit: since it's used only in `impl error::Error` using `impl std::error::Error` would save this two lines

Reply:
Does not work on 1.29 :(

 
error[E0658]: access to extern crates through prelude is experimental (see issue #44660)
   --> src/util/ecdsa.rs:471:6
    |
471 | impl std::error::Error for EcdsaSigError {}